src/components/TopBar.js

```diff
@@ -215,14 +215,14 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
               href: Nav.getLink('library-datasets'),
               onClick: hideNav
             }, ['Data']),
+            isDataBrowserVisible() && h(DropDownSubItem, {
+              href: Nav.getLink('library-browser'),
+              onClick: hideNav
+            }, ['Try the BETA Data Catalog']),
```

Comment:
```suggestion
            }, ['Data Catalog (BETA)']),
```

A phrase here seems out of place since the other options are all nouns.

Reply:
I want to stick with the mock ups UX gave us for this wording, and revisit it later. Right now, at least, this feature is not ready to be turned on anyway, so there will be things we need to do before that too